dependencies.gradle

```diff
@@ -117,7 +118,8 @@ ext.libraries = [
         recyclerAnimators    : "jp.wasabeef:recyclerview-animators:$versions.recyclerAnimators",
         materialProgress     : "me.zhanghai.android.materialprogressbar:library:$versions.materialProgress",
         circleImageView      : "de.hdodenhof:circleimageview:$versions.circleImageView",
-        bottomNavigationView : "com.github.ittianyu:BottomNavigationViewEx:$versions.bottomNavigationView",
+        shortcutBadger       : "me.leolin:ShortcutBadger:$versions.shortcutBadger",
```

Comment:
`shortcutBadger` is useful thing and I have no idea how it can be replaced with minimum effort. So, it can be moved from list, that called  _Candidates for removal_